namespaces/init.go

```diff
@@ -95,7 +103,11 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 
 	label.Init()
 
-	if err := mount.InitializeMountNamespace(rootfs,
+	if (cloneFlags & syscall.CLONE_NEWNS) == 0 {
```

Comment:
@LK4D4 unfortunately it is required here, because InitializeMountNamespace() has side-effects even if a config doesn't contain MountConfig. The first "else" was added just to have the same style in all places.